components/search/SearchForm.vue

```diff
@@ -116,7 +116,7 @@
           path: this.localePath({
             name: 'search'
           }),
-          query: { ...this.$route.query, page: 1 }
+          query: { ...this.$route.query, page: 1, api: undefined }
```

Comment:
Ensure that the `api` query parameter is removed when leaving the newspapers collection entity page by clicking the `x` in the search pill.